css/styles.css

```diff
@@ -745,7 +745,7 @@
 }
 
 .more-replies > .number-of-replies {
```

Comment:
Kindly make use of the '_.number-of-replies_' class only, if there's not another '_.number-of-replies_' class in the CSS file, there will be no problem with using that class. This is to avoid ambiguity and to have a clean code understandable to other people like your coding partners.